script.js

```diff
@@ -1,4 +1,586 @@
+  var randomInteger = Math.floor(randomDecimal);
+  if (randomInteger == 0) {
+    assignedChoice = `scissors`;
+  }
+  if (randomInteger == 1) {
+    assignedChoice = `paper`;
+  }
+  if (randomInteger == 2) {
+    assignedChoice = `stone`;
+  }
+  return assignedChoice;
+};
+
+// Create a function to derive the result of a standard SPS game
+var result = function (userInput, programSelection) {
+  var gameResult = ``;
+  if (userInput == programSelection) {
+    gameResult = `draw`;
+  }
+  if (
+    (userInput == `scissors` && programSelection == `paper`) ||
+    (userInput == `paper` && programSelection == `stone`) ||
+    (userInput == `stone` && programSelection == `scissors`)
+  ) {
+    gameResult = `win`;
+  }
+  if (
+    (userInput == `scissors` && programSelection == `stone`) ||
+    (userInput == `stone` && programSelection == `paper`) ||
+    (userInput == `paper` && programSelection == `scissors`)
+  ) {
+    gameResult = `lose`;
+  }
+  console.log(`user input ${userInput}`);
+  console.log(`program selection ${programSelection}`);
+  console.log(`game result ` + gameResult);
+  return gameResult;
+};
+
+// Create a function for standard game mode
+var standardGame = function (userInput, programSelection) {
+  var gameResult = result(userInput, programSelection);
+  var message = `Sorry, I didn't get that. <br> <br> Please type 'scissors', 'paper' or 'stone' to start playing the game with me 😊 <br> <br> You can also type 'korean' to try our korean mode and 'standard' for the original game. <br> <br> --- What is the korean version of SPS? --- <br> In Korea, a two-player upgraded version exists by the name muk-jji-ppa. After showing their hands, the player with the winning throw shouts "muk-jji-ppa!" upon which both players throw again. If they throw differently (for example, rock and paper, or paper and scissors), whoever wins this second round shouts "muk-jji-ppa!" and thus the play continues until both players throw the same item (for example, rock and rock), at which point whoever was the last winner becomes the actual winner.`;
+
+  if (userInput == `standard`) {
+    message = `You are now in the standard game mode. <br> <br> Please type 'scissors', 'paper' or 'stone' to start the standard SPS game with me!`;
+  }
+  if (gameResult == `draw`) {
+    message = `It's a draw! <br> <br> This is the standard game. <br> <br> We both chose ${programSelection}! <br> You didn't copy my answer did you? 😜`;
+  }
+  if (gameResult == `win`) {
+    message = `You won! <br> <br> This is the standard game. <br> <br> You chose ${userInput} and I chose ${programSelection}. <br> Good job!`;
+  }
+  if (gameResult == `lose`) {
+    message = `You lost! <br> <br> This is the standard game. <br> <br> You chose ${userInput} and I chose ${programSelection}. <br> Better luck next time!`;
+  }
+  return message;
+};
+
+// Create a function to update winning player for the last round
+var updateWinner = function (userInput, programSelection) {
```

Comment:
It is great that you are keeping the naming of your parameters - `useInput, programSelection` for your relevant functions consistent. 👍